_variables.yml

```
@@ -1,17 +1,5 @@
 design:
-  data-resource: |
-    This is the software component that
-    manages and stores data from a research study or data source. For
-    example, our external partner DD2 (listed below in the Stakeholder
-    section) collects a wide variety of data on a set of participants.
-    The data generated from these participants enrolled with DD2 would
-    form a single "Data Resource".
-  whitebox: |
-    A description of the internal functioning and behaviour of a system.
-    For example, how the internals of the Seedcase framework like the web frontend 
-    part, the programmatic API part, and the data backend environment part interact
-    with each other. "White" implying seen or visible (e.g. daylight).
-  blackbox: |
+  black-box: |
```

```suggestion
  closed-box: |
```

To use more inclusive (and more explicit) language.